tools/ebpf/trylock-stats/trylock-stats.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+#
+#  bcc tool to observe pthread mutex trylock
+#
+#  Based on https://github.com/goldshtn/linux-tracing-workshop/blob/master/lockstat-solution.py
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+import subprocess
+import sys
+import itertools
+from time import sleep
+from bcc import BPF
+from optparse import OptionParser
```

Comment:
`optparse` is deprecated since Python 3.2. Better to use `argparse`. (It should be pretty easy to convert to `argparse`).

Reply:
These C and Python code from original code, but it should be fixed. Thanks for checking!